packages/client/src/object/convertWireToOsdkObjects.ts

```diff
@@ -165,14 +222,15 @@ export async function convertWireToOsdkObjectsInPlace(
   );
 
   for (const obj of objs) {
-    const objectDef = await localObjectCache.get(client, obj.$apiName);
+    const objectDef: ObjectOrInterfaceDefinition = await localObjectCache.get(
```

Comment:
This is still loading object definitions in serial as you traverse the objs array.

edit: oh it seems like you plan on only calling this with a single object type?

Reply:
I forget if this PR or the followup did this but its okay at the end of the chain.